daprdocs/content/en/operations/components/pluggable-components-registration.md

```diff
@@ -49,49 +49,43 @@ Both your component and the Unix Socket must be running before Dapr starts.
 
 By default, Dapr looks for [Unix Domain Socket][uds] files in the folder in `/tmp/dapr-components-sockets`.
 
-Filenames in this folder are significant for component registration. They must be formed by appending the component's name with a file extension of your choice, more commonly `.sock`. For example, the filename `my-component.sock` is a valid UDS file name for a component named `my-component`.
+Filenames in this folder are significant for component registration. They must be formed by appending the component's **name** with a file extension of your choice, more commonly `.sock`. For example, the filename `my-component.sock` is a valid UDS file name for a component named `my-component`.
 
 Since you are running Dapr in the same host as the component, verify this folder and the files within it are accessible and writable by both your component and Dapr.
 
-## Define the component
+### Building-blocks discovery from  behind the same component
```

Comment:
We could probably rename this for section to "building block discovery and multiplexing"

Reply:
While I understand using "building block" to differentiate from "component", and could live with it, I still feel it's a bit imprecise.  To me, "component discovery and multiplexing" feels most accurate.

Reply:
I am happy with that :)

```suggestion
### component discovery and multiplexing
```